### ISSUE TITLE
why bone conduction speech have noise at 2000Hz

### ISSUE THREAD

**Issue by andyye1999** [wangmou21/abcs#1]

**Comment by jhauret** [comment]
There is indeed more energy around 2kHz as shown by this spectrogram.
![bcm_spectrogram](https://user-images.githubusercontent.com/53187038/225921507-fbd87fa4-0d93-4c0c-89f8-7a7ce3638a6f.png)

My take is that it is due to the bone conduction microphone's frequency response which is not flat. Indeed, when the speaker is not talking, this noise disappears. Could you share this curve if you have it or the exact reference of the SabineTek bone-conducted microphone?

**Comment by lixinghe1999** [comment]
> 

according to the spectrogram, seems the noise is random, not exactly appear when the speaker is not talking.

**Comment by jhauret** [comment]
Indeed, you are correct in pointing out the presence of a random noise, possibly pink in nature, that spans across the entire frequency spectrum, with a greater concentration towards lower frequencies. However, my initial statement was specifically regarding the energy line at 2kHz, which exhibits a more pronounced intensity in regions associated with active speech. Upon revisiting the article, the authors provide an explanation for this phenomenon, which states:
_"The low-frequency self-noise was essentially generated by the accidental friction between the BC microphone and the
skin of the speaker. The sustained narrow-band noise was generated by the resonance of the BC microphone and the contact skin of the speaker."_
